### PR TITLE
Add tdbg import workflow command

### PR DESCRIPTION
**Pull request by wxing1292**
**What changed?**
* Add tdbg import workflow command

**Why?**
more admin API

**How did you test it?**
manual test

**Potential risks**
N/A

**Is hotfix candidate?**
N/A

### CODE REVIEW

**Comment by wxing1292** [comment]
review #4850 first

File: tools/tdbg/commands.go [hunk 1]
```diff
@@ -121,13 +128,115 @@ func AdminShowWorkflow(c *cli.Context) error {
...
 		}
 		if err := os.WriteFile(outputFileName, data, 0666); err != nil {
-			return fmt.Errorf("unable to export History data file: %s", err)
+			return fmt.Errorf("unable to write History data file: %s", err)
+		}
+	}
+	return nil
+}
+
+// AdminImportWorkflow shows history
```
Comment (yux0) [inline tools/tdbg/commands.go:142]: nit: imports workflow history?

File: tools/tdbg/commands.go [hunk 2]
```diff
@@ -121,13 +128,115 @@ func AdminShowWorkflow(c *cli.Context) error {
...
+func AdminImportWorkflow(c *cli.Context) error {
+	nsName, err := getRequiredOption(c, FlagNamespace)
+	if err != nil {
+		return err
+	}
+	wid, err := getRequiredOption(c, FlagWorkflowID)
+	if err != nil {
+		return err
+	}
+	rid, err := getRequiredOption(c, FlagRunID)
```
Comment (yux0) [inline tools/tdbg/commands.go:152]: Is this required? Will the history contains the run id?

File: service/history/handler.go [hunk 3]
```diff
@@ -742,6 +742,9 @@ func (h *Handler) ImportWorkflowExecution(ctx context.Context, request *historys
 		return nil, h.convertError(err)
 	}
 
+	if !shardContext.GetClusterMetadata().IsGlobalNamespaceEnabled() {
```
Comment (yux0) [inline service/history/handler.go:745]: Anything will break if the global namespace is disabled?

File: tools/tdbg/commands.go (hunk 2 above)
Comment (wxing1292) [reply tools/tdbg/commands.go:152]: this is required, the history will only contain next / prev run ID.

i also believe the next / prev run ID should be excluded from history

File: service/history/handler.go (hunk 3 above)
Comment (wxing1292) [reply service/history/handler.go:745]: nil pointer exception, plus ndc related logic may not be initialized

File: tools/tdbg/commands.go [hunk 4]
```diff
@@ -48,6 +50,11 @@ import (
 	"go.temporal.io/server/common/primitives/timestamp"
 )
 
+const (
+	historyImportBlobSize = 16         // 256K
```
Comment (yycptt) [inline tools/tdbg/commands.go:54]:
```suggestion
	historyImportBlobSize = 16
```

File: tools/tdbg/commands.go [hunk 5]
```diff
@@ -94,13 +101,13 @@ func AdminShowWorkflow(c *cli.Context) error {
 			NextPageToken:     token,
 		})
 		if err != nil {
-			return fmt.Errorf("unable to read History Branch: %s", err)
+			return fmt.Errorf("unable to recv History Branch: %s", err)
```
Comment (yycptt) [inline tools/tdbg/commands.go:104]: is this change intentional?

**Review by yycptt — APPROVED** [review]

File: tools/tdbg/commands.go (hunk 5 above)
Comment (wxing1292) [reply tools/tdbg/commands.go:104]: yes
